server/models/user.model.js

```diff
@@ -54,6 +54,13 @@ const UserSchema = new mongoose.Schema(
       select: false,
     },
     phone: String,
+    role: {
+      type: Number,
+      default: 1,
+    },
+    assignedProperties: {
+      type: [],
```

Comment:
This should be a list of ObjectIds. Something like this (You might need to confirm from mongoose documentation):
```suggestion
      type: [{ type : ObjectId, ref: 'Property' }],
```